gguf-py/gguf/constants.py

```diff
@@ -850,6 +850,7 @@ def get_type(val: Any) -> GGUFValueType:
     GGMLQuantizationType.IQ2_XS:  (256, 2 + QK_K // 4 + QK_K // 32),
     GGMLQuantizationType.IQ3_XXS: (256, 2 + QK_K // 4 + QK_K // 8),
     GGMLQuantizationType.IQ1_S:   (256, 2 + QK_K // 8 + QK_K // 16),
+    GGMLQuantizationType.IQ1_M:   (256, QK_K // 8 + QK_K // 16  + QK_K // 32),
```

Comment:
Move it after `F64` down below, to follow the order of the enum:

https://github.com/ggerganov/llama.cpp/blob/8b1b1f4982d3e9b994308d05a1c8b9e45c23edb5/gguf-py/gguf/constants.py#L783-L791

Reply:
I've moved it after `F64`